.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -28,14 +28,21 @@ For refactoring and code cleanup changes, exercise the code before and after the
 
 ### Proposed changelog entries
 
-- Entry 1: Issue, human-readable text
-- […]
-
-<!-- Comment:
+<!--
+Please propose a changelog entry (or entries) for this pull request below between the lines with three backticks.
```

Comment:
As an alternative, should we tell them to replace the specific text with their own text?
```suggestion
Please propose a changelog entry (or entries) for this pull request by replacing the 'Improve spline reticulation' text below with the changelog entry for the pull request.
```